src/utils/events/move-input.js

```diff
@@ -25,12 +25,14 @@ export default class MoveInput {
     this.options.events.forEach(event => this.element.removeEventListener(event, this.handleEvent));
   }
 
-  set(options) {
-    Object.assign(this.options, options);
-  }
-
-  isSourceOf(eventName) {
-    return MOVE_EVENT_TYPES.indexOf(eventName) >= 0;
+  /**
+   * Enable this input (begin processing events)
+   * if the specified event type is among those handled by this input.
+   */
+  enableOnEventType(eventType) {
```

Comment:
I can't find a good name for this function. We're not enabling for a specific event type, we're enabling the entire input if the passed event type is one that the input handles.

As for disabling, yes, I suppose we might need to (though I'd say it's fairly unlikely); I added @Pessimistress' `set()` method back in to support that.